src/com/projectkorra/projectkorra/waterbending/plant/PlantRegrowth.java

```diff
@@ -138,4 +112,12 @@ public void setBlock(final Block block) {
 		this.block = block;
 	}
 
+	public TempBlock getPlant1() {
+		return bottomPlant;
+	}
+
+	public void setPlant1(TempBlock plant1) {
+		this.bottomPlant = plant1;
+	}
+
```

Comment:
These aren't very descriptive names, might be worth changing to `getBottomPlant` and `setBottomPlant` or something?

Reply:
Yeah I autogenerated these getters & setters when I still named it plant1 and plant2, will fix.